lib/skate/detours/snapshot_serde.ex

```diff
@@ -82,17 +87,16 @@ defmodule Skate.Detours.SnapshotSerde do
 
   defp validate_serialized_snapshot(%Detour{id: id} = detour) do
     serialized_snapshot = serialize_snapshot(detour)
-    {matches, diff, scoped_state} = compare_snapshots(detour, serialized_snapshot)
+    {matches, diff} = compare_snapshots(detour, serialized_snapshot)
 
     if matches do
       serialized_snapshot
     else
-      scoped_state = fix_snapshot_activated_at(scoped_state, detour)
       Sentry.capture_message(mismatch_message(id), extra: %{diff: diff})
 
       Logger.error("#{mismatch_message(id)} #{diff_details(diff)}")
 
-      scoped_state
```

Comment:
Just confirming: if the serialized-from-db-data detour does not equal the original snapshot we have for that detour, we're falling back to the serialized version? That's a divergence from the original intention of the validation logic. It's not necessarily a wrong change, I'm just trying to figure out which one is the right approach, and I don't know

Reply:
ok got it!! Perfect explanation, thank you